cabal-ghc921.project

```diff
@@ -11,20 +11,44 @@ packages:
         --  ./plugins/hls-stylish-haskell-plugin
         --  ./plugins/hls-fourmolu-plugin
          ./plugins/hls-class-plugin
-         ./plugins/hls-eval-plugin
+        -- ./plugins/hls-eval-plugin
          ./plugins/hls-explicit-imports-plugin
          ./plugins/hls-refine-imports-plugin
-         ./plugins/hls-hlint-plugin
+        -- ./plugins/hls-hlint-plugin
+        ./plugins/hls-rename-plugin
         -- ./plugins/hls-retrie-plugin
-         ./plugins/hls-haddock-comments-plugin
-        --  ./plugins/hls-splice-plugin
+        -- ./plugins/hls-haddock-comments-plugin
+        -- ./plugins/hls-splice-plugin
          ./plugins/hls-qualify-imported-names-plugin
-         ./plugins/hls-floskell-plugin
+        -- ./plugins/hls-floskell-plugin
          ./plugins/hls-pragmas-plugin
          ./plugins/hls-module-name-plugin
-         ./plugins/hls-ormolu-plugin
+        -- ./plugins/hls-ormolu-plugin
          ./plugins/hls-call-hierarchy-plugin
-         ./plugins/hls-alternate-number-format-plugin
+        -- ./plugins/hls-alternate-number-format-plugin
+
+source-repository-package
+  type: git
+  location: https://github.com/tfausak/unix-compat
+  tag: 154c3a63f154cb49c51d5f9d13488e8119631d8a
+  -- To fix windows build
+  -- https://github.com/jacobstanley/unix-compat/pull/47
+
+source-repository-package
+  type: git
+  location: https://github.com/Bodigrim/th-extras
```

Comment:
`lsp` upgrade dropped the dependency here, I don't think we depend on it anywhere else, so I *think* we can ditch this.